its/src/org/sonarlint/eclipse/its/reddeer/views/SonarLintConsole.java

```diff
@@ -70,6 +70,18 @@ public void enableAnalysisLogs() {
     }
   }
 
+  public void enableIdeSpecificLogs(boolean enabled) {
+    consoleView.activate();
+    var menu = new ToolItemMenuItem(new DefaultToolItem(consoleView.getCTabItem().getFolder(), "Configure logs"), "IDE-specific traces");
+
+    // When the IDE-specific logs are not selected but we want to enable them, we "select" the menu item.
+    // When the IDE-specific logs are selected but we want to disable them, we "de-select" the menu item.
+    // -> `menu.select()` is used for toggling them on/off, not the greatest naming.
+    if ((!menu.isSelected() && enabled) || (menu.isSelected() && !enabled)) {
```

Comment:
Is the 'if statement' even needed? maybe the comment above is enough

Reply:
Yes it is, so we don't accidentily trigger the menu item on or off. As I said it can be both selecting and de-selecting.
This if statement is used so we only select it if we really want to have it enabled (if already enabled, nothing needs to be done) and we de-select it if we want to have it disabled (if already disabled, nothing needs to be done).